package.json

```diff
@@ -2,7 +2,7 @@
   "name": "archit.us",
   "private": true,
   "scripts": {
-    "start": "react-static start",
+    "start": "react-static start --open",
```

Comment:
what does this do?

Reply:
`--host=0.0.0.0` it shouldn't have been commited 😊